azurelinuxagent/common/protocol/extensions_goal_state_from_vm_settings.py

```diff
@@ -142,11 +142,7 @@ def extensions(self):
         return self._extensions
 
     def get_redacted_text(self):
-        return ExtensionsGoalStateFromVmSettings.redact(self._text)
-
-    @staticmethod
-    def redact(text):
-        return re.sub(r'("protectedSettings"\s*:\s*)"[^"]+"', r'\1"*** REDACTED ***"', text)
```

Comment:
The redact() method was used in a previous iteration of the code, but now there is no need for it, so I folded it into get_redacted_text()